keyboards/splitkb/aurora/sofle_v2/rev1/config.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2023 splitkb.com <support@splitkb.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// `SPLIT_HAND_PIN` can currently be defined in `info.json`,
+// but can't yet be given a value
+#define SPLIT_HAND_PIN F4
+
+// Not yet available in `info.json`
+#ifdef OLED_ENABLE
+#    define OLED_DISPLAY_128X32
+#    define SPLIT_OLED_ENABLE
+#endif
+
+// Not yet available in `info.json`
+#ifdef RGB_MATRIX_ENABLE
+#    define RGB_MATRIX_LED_COUNT RGBLED_NUM
+#    define RGB_MATRIX_SPLIT RGBLED_SPLIT
+#    define RGB_MATRIX_MAXIMUM_BRIGHTNESS 128
+#    define SPLIT_TRANSPORT_MIRROR
+#endif
+
+// Not yet available in `info.json`
+#ifdef BOOTMAGIC_ENABLE
+     // Top left key on left half
+#    define BOOTMAGIC_LITE_ROW 0
+#    define BOOTMAGIC_LITE_COLUMN 5
+     // Top right key on right half
+#    define BOOTMAGIC_LITE_ROW_RIGHT    5
+#    define BOOTMAGIC_LITE_COLUMN_RIGHT 5
+#endif
```

Comment:
These can go to info.json.

Reply:
RGB_MATRIX_LED_COUNT is not yet available in info.json as far as I can see?